sympy/physics/quantum/tests/test_printing.py

```diff
@@ -37,8 +34,17 @@
 
 MutableDenseMatrix = Matrix
 
+
 ENV = {}
-exec_("from sympy import *", ENV)
+exec_('from sympy import *', ENV)
+exec_('from sympy.physics.quantum import *', ENV)
+exec_('from sympy.physics.quantum.cg import *', ENV)
+exec_('from sympy.physics.quantum.spin import *', ENV)
+exec_('from sympy.physics.quantum.hilbert import *', ENV)
+exec_('from sympy.physics.quantum.qubit import *', ENV)
+exec_('from sympy.physics.quantum.qexpr import *', ENV)
+exec_('from sympy.physics.quantum.gate import *', ENV)
+exec_('from sympy.physics.quantum.constants import *', ENV)
```

Comment:
I've added a number of things to `__all__` in quantum sub-packages so that the star-import environment above is sufficient for `srepr/eval` round-tripping.